pubspec.yaml

```
@@ -39,7 +39,8 @@ dependencies:
   intl: ^0.20.2
   json_annotation: ^4.9.0
   go_router: ^14.8.1
-
+  get_it:
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)